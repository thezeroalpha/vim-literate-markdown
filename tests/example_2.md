# Example 2
Example 2 from the Vim documentation.

<!-- :Tangle DIRNAME/FILENAME.out -->
<!-- :Tangle(python) DIRNAME/FILENAME.py -->

```ruby
puts "Sample text"
```

```python
print("Sample text")
```

```
this is some pseudocode in a block with no language
```

```bash
echo "Sample text"
```
