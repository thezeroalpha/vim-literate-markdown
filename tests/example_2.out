puts "Sample text"

echo "Sample text"

